pwiz_tools/Skyline/TestRunner/TestRunner.csproj

```diff
@@ -60,7 +60,7 @@
     <OutputPath>..\bin\x64\Debug\</OutputPath>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <DebugType>full</DebugType>
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <PlatformTarget>x64</PlatformTarget>

```

Comment:
And not sure how this would impact running 32-bit tests with TestRunner.

Reply:
Shouldn't affect it at all. It only sets x64 for the 64-bit TestRunner exe. 32-bit tests will run the 32-bit TestRunner.exe.